request.go

```diff
@@ -117,6 +117,22 @@ func (r *Request) SetHeaders(headers map[string]string) *Request {
 	return r
 }
 
+// SetMultiValueHeaders sets multiple headers fields and its values is list of strings at one go in the current request.
+//
+// For Example: To set `Accept` as `text/html, application/xhtml+xml, application/xml;q=0.9, image/webp, */*;q=0.8`
+//
+// 		client.R().
+//			SetMultiValueHeaders(map[string][]string{
+//				"Accept": []string{"text/html", "application/xhtml+xml", "application/xml;q=0.9", "image/webp", "*/*;q=0.8"},
+//			})
+// Also you can override header value, which was set at client instance level.
+func (r *Request) SetHeaderMultiValues(headers map[string][]string) *Request {
```

Comment:
@mhdiiilham @rizalgowandy FYI, godoc is not updated with the updated method name. I will be doing that. Thanks.

Reply:
oh yeah, I'm sorry I missed the godoc comment. Thank you @jeevatkm

Reply:
@mhdiiilham No worries, I just did. Maybe next time 👍